BREEZE.rst

```diff
@@ -1947,6 +1951,11 @@ This is the current syntax for  `./breeze <./breeze>`_:
 
                  cassandra kerberos mongo openldap presto rabbitmq redis all
 
+  --start-airflow
```

Comment:
Most other options like this area command, not a flag, so this should be `breeze start-airflow` for consistency I think

Reply:
I had exactly the same thought initially, but after a bit of thought - I look at this as an option of "shell" command (this is the default command when none is specified)  -  it is much more similar to `--db-reset` flag, `--db-reset` means "reset DB when you enter the shell" and `--start-airflow` means "start airflow when you enter the shell.

Reply:
I see your point, it makes sense for me.
How should I proceed?

Reply:
But when I think about it now - this might be looked at s indeed pretty "prominent" ways of using Airflow so it might deserve it's own command WDYT @ashb @turbaszek @francescomucio ?

Reply:
My thinking was that since this doesn't couple work _any_ of the existing commands (I.e. `breeze _anything_ --start-airflow` doesn't make sense) that it is a command, not a flag.

Reply:
> But when I think about it now - this might be looked at as indeed pretty "prominent" way of using Airflow so it might deserve it's own command WDYT @ashb @turbaszek @francescomucio ?

We shouldn't direct most end users to this (as it stands) - as it will run main branch, not a released version

Reply:
OK some guidelines then @francescomucio  if you want to give a shot at that -> look how "shell is defined for example - it should set the "command_to_run" to a new command ("start_airflow"?) and then you can look for "enter_breeze" command and implement it similarly - but only for CI image (there is no tmux in prod image).

Reply:
> We shouldn't direct most end users to this (as it stands) - as it will run main branch, not a released version

True. The "prominent" in this case means being a separate command rather than option. Do you think it's better to leave it as a  `shell` option then? For me - I can go with both, but --start-airflow does indicate a bit more "developer'y" option rather than "first class command".

Reply:
Yeah, not sure anymore either :)

Reply:
Let's merge it as is - we can change later.

Reply:
thanks for your support guys, it was fun for me to work on this :)